lib/sedna/service/client.py

```diff
@@ -141,15 +143,18 @@ def __init__(self, url, client_id, **kwargs):
             "ping_interval": interval,
             "max_size": min(max_size, 16 * 1024 * 1024)
```

Comment:
this constant may put in `constant.py`

Reply:
This may not be appropriate to use a constant instead of the value